### DISCUSSION TITLE
Can Analyzer Properties be automated?

### DISCUSSION THREAD

**Discussion by JayMartin0718** [Riverside-Software/sonar-openedge#963] [Q&A] [answered]
For example, I currently have to manually enter a value for sonar.oe.databases. Is there a file this information is stored in that I could have a script update during install?

**Comment by gquerret** [comment, marked as answer]
SonarScanner properties are read by default in a file called `sonar-project.properties` ([documentation](https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/)). They can be overidden on the command line.
